src/all_types.hpp

```diff
@@ -648,6 +648,7 @@ struct AstNodeFnCallExpr {
     ZigList<AstNode *> params;
     bool is_builtin;
     bool is_async;
+    bool seen; // used by @compileLog
```

Comment:
Let's just merge this as is, but one idea would be to rename this `seen_by_compile_log` and then the comment would be unnecessary.

Reply:
Ah, I would have pushed again for that, but thanks anyways.